src/sentry/backup/dependencies.py

```diff
@@ -701,3 +701,28 @@ def get_exportable_sentry_models() -> set[type[models.base.Model]]:
             get_final_derivations_of(BaseModel),
         )
     )
+
+
+def merge_users_for_model_in_org(
+    model: type[models.base.Model], *, organization_id: int, from_user_id: int, to_user_id: int
+) -> None:
+    """
+    All instances of this model in a certain organization that reference both the organization and
+    user in question will be pointed at the new user instead.
+    """
+
+    from sentry.models.organization import Organization
+    from sentry.models.user import User
+
+    model_relations = dependencies()[get_model_name(model)]
+    user_refs = {k for k, v in model_relations.foreign_keys.items() if v.model == User}
+    org_refs = {
+        k if k.endswith("_id") else f"{k}_id"
```

Comment:
Why do we need to check for `_id` in the name for org but not user?

Reply:
Because user fields are basically just ints already ("HybridCloudForeignKey"), whereas `FlexibleForeignKey` is more convoluted and makes a `foo` and `foo_id` field for each such relation. Since all orgs are `FlexibleForeignKey`s here (we are in the region silo for this function), we want to make sure we get the `int`.